utils/defillama.ts

```diff
@@ -1,29 +1,75 @@
 import axios from "axios";
 
 const baseUrl = "https://api.llama.fi/protocol/uma-protocol";
+const defillamaUrls: string[] = [
+  "https://api.llama.fi/protocol/polymarket",
+  "https://api.llama.fi/protocol/sherlock",
+  "https://api.llama.fi/protocol/jarvis-network",
+  "https://api.llama.fi/protocol/across",
+  "https://api.llama.fi/protocol/uma-protocol",
+];
 
 interface TvlData {
   date: string;
   totalLiquidityUSD: number;
 }
 
-export async function getDefillamaTvl(url: string = baseUrl) {
-  const { data } = await axios.get(url);
-  const defillamaTvlData: TvlData[] = data.tvl.slice(-1);
-  const latestTvl = defillamaTvlData[0].totalLiquidityUSD;
+export async function getDefillamaTvl() {
```

Comment:
would hvae really preferred you leave the old function in, add a new one that calls it with the array of urls and does the sum separately. kind of keeps old functionality while using it to build the new feature.

Reply:
I'm with David on this. Keeping the api calls and the calculations separately would be nice. If it's urgent you can merge it as it is, but we'll refactor it at a later time when we touch this code base again

Reply:
Sorry about that! I don't want to cause more work for you later. I just made an update trying to incorporate your suggestion. Let me know if it looks fine and I will merge.